### PR TITLE
Responsive ad - expandable "show more" button

### PR DESCRIPTION
**Pull request by austinh7**
This adds a button to the bottom of the expandable, when clicked it slides out of view.

It will only apply to the new template version of expandables, (currently there aren't any running.)

expandableopenclose

### CODE REVIEW

File: static/src/stylesheets/module/commercial/_creatives.scss [hunk 1]
```diff
@@ -250,9 +250,43 @@
...
-    
+
+    .ad-exp__open-chevron {
+        position: absolute;
+        z-index: 4;
+        bottom: 0;
+        left: 50%;
+        border: 1px solid #ffffff;
+        border-width: 1px 1px 0;
+        background: #000000;
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/commercial/_creatives.scss]: Pure black?

Comment (austinh7) [inline static/src/stylesheets/module/commercial/_creatives.scss]: It has 50% transparency on it

File: static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v2.js [hunk 2]
```diff
@@ -50,10 +53,18 @@ define([
     };
 
     ExpandableV2.prototype.create = function () {
-        var $expandablev2 = $.create(template(expandableV2Tpl, this.params));
+
+        var $expandablev2, showmore;
+
+        showmore = {
```
Comment (ironsidevsquincy) [inline static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v2.js]: combine this with the decleration above

File: static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v2.js [hunk 3]
```diff
@@ -50,10 +53,18 @@ define([
...
+            show: (this.params.showMore === 'yes') ?
+                '<button class="ad-exp__open-chevron ad-exp__open"><i class="i i-arrow-white-down-36"></i></button>' : ''
+        };
+
+        $expandablev2 = $.create(template(expandableV2Tpl, merge(this.params, showmore)));
```
Comment (ironsidevsquincy) [inline static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v2.js]: and this

File: static/src/stylesheets/module/commercial/_creatives.scss [hunk 4]
```diff
@@ -250,9 +250,37 @@
...
-    
+
+    .ad-exp__open-chevron {
+        position: absolute;
+        z-index: 4;
+        bottom: 0;
+        left: 50%;
+        border: 1px solid #ffffff;
+        border-width: 1px 1px 0;
+        background: #333333;
```
Comment (ironsidevsquincy) [inline static/src/stylesheets/module/commercial/_creatives.scss]: this isn't one of our colours?

File: static/src/stylesheets/module/commercial/_creatives.scss [hunk 5]
```diff
@@ -250,9 +250,37 @@
...
+        color: #ffffff;
+        opacity: .5;
+        padding: $gs-baseline/2 0 0;
+        width: $gs-gutter*3;
```
Comment (ironsidevsquincy) [inline static/src/stylesheets/module/commercial/_creatives.scss]: should this be a `gs-span`

File: static/src/stylesheets/module/commercial/_creatives.scss (hunk 4 above)
Comment (sammorrisdesign) [inline static/src/stylesheets/module/commercial/_creatives.scss]: That's some classic `neutral-1` right there

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**